integration-cli/docker_cli_commit_test.go

```diff
@@ -49,10 +50,9 @@ func (s *DockerSuite) TestCommitPausedContainer(c *check.C) {
 	out, _ = dockerCmd(c, "commit", cleanedContainerID)
 
 	out, err := inspectField(cleanedContainerID, "State.Paused")
-	c.Assert(err, check.IsNil)
-	if !strings.Contains(out, "true") {
-		c.Fatalf("commit should not unpause a paused container")
-	}
+	c.Assert(err, checker.IsNil)
+	c.Assert(out, checker.Contains, "true",
+		check.Commentf("commit should not unpause a paused container"))
```

Comment:
No need for the new line
Could also use a `// commit should not unpause a paused container` above the assert rather than using a Commentf.